Makefile

```diff
@@ -76,11 +76,12 @@ goword:tools/bin/goword
 gosec:tools/bin/gosec
 	tools/bin/gosec $$($(PACKAGE_DIRECTORIES))
 
-check-static:tools/bin/gometalinter
-	@ # vet and fmt have problems with vendor when ran through metalinter
-	tools/bin/gometalinter --disable-all --deadline 120s \
+check-static:tools/bin/gometalinter tools/bin/misspell tools/bin/ineffassign
+	@ # TODO: enable megacheck.
```

Comment:
Because `kv/memdb_buffer.go:71:41:warning: cannot use &(util.Range literal) (value of type *util.Range) as *util.Range value in argument to m.db.NewIterator (megacheck)` 
`meagacheck` is disabled.